mmeds/util.py

```diff
@@ -702,6 +703,11 @@ def send_email(toaddr, user, message='upload', testing=False, **kwargs):
                'the following access code:\n{code}\n\nBest,\nMmeds Team\n\n' +\
                'If you have any issues please email: {cemail} with a description of your problem.\n'
         subject = 'New Data Uploaded'
+    elif message == 'upload-run':
```

Comment:
you might want to store the messages in a data structure somewhere, instead of directly inserting them here.